src/store/plugins/names.js

```diff
@@ -116,7 +112,10 @@ export default (store) => {
         commit,
       }) {
         commit('setAreNamesFetching', true);
-        const fetchPendingNameClaimTransactions = (address) => fetchPendingTransactions(address)
+        const { fetchPendingTransactions } = ProtocolAdapterFactory.getAdapter(PROTOCOL_AETERNITY);
```

Comment:
In my opinion picking methods from a class by destructuring it is rather bad pattern. I suggest to save the adapter instance to constant:
```
const adapter = ProtocolAdapterFactory.getAdapter(PROTOCOL_AETERNITY);
```
and then use methods on it:
```
adapter.someMethod()
```